nav2_behavior_tree/CMakeLists.txt

```diff
@@ -111,6 +111,9 @@ list(APPEND plugin_libs nav2_truncate_path_action_bt_node)
 add_library(nav2_goal_updater_node_bt_node SHARED plugins/decorator/goal_updater_node.cpp)
 list(APPEND plugin_libs nav2_goal_updater_node_bt_node)
 
+add_library(nav2_planner_selector_bt_node SHARED plugins/decorator/planner_selector_node.cpp)
```

Comment:
Update readme table to include this BT node

Reply:
Should I add a new section to explain the selection of the planner (and later of the controller)? something like "Selecting your planner and controller from ros topics"

I do not see clearly this behavior should got into the Action section (even if it is in the action folder).

How do I proceed here?

Reply:
https://github.com/ros-planning/navigation2/tree/main/nav2_behavior_tree#navigation-specific-behavior-tree-nodes simply add a new entry into the table.